### PR TITLE
🐛 Fix CoinGecko Invalid Hook Call Error

### PR DESCRIPTION
**Pull request by kacperkurek**

### CODE REVIEW

File: packages/coingecko/package.json [hunk 1]
```diff
@@ -38,8 +38,10 @@
     "@ethersproject/contracts": "^5.6.0",
     "@ethersproject/providers": "^5.6.2",
     "@usedapp/core": "workspace:*",
-    "isomorphic-fetch": "^3.0.0",
-    "react": "^17.0.1"
+    "isomorphic-fetch": "^3.0.0"
```
Comment (mj426382) [inline packages/coingecko/package.json:41]:
```suggestion
    "isomorphic-fetch": "^3.0.0",
    "react": "^17.0.1"
```

Comment (mj426382) [reply packages/coingecko/package.json:41]: Add react to depedencies and pnpm install again.

Comment (kacperkurek) [reply packages/coingecko/package.json:41]: I have app using react 18 and your suggestion will cause Invalid Hook Call Error

Comment (kacperkurek) [reply packages/coingecko/package.json:42]: added to devDependencies

**Comment by rzadp** [comment]
@kacperkurek Thanks you so much :) But could you have not included the linting changes in this PR, to limit changes under review?

**Comment by kacperkurek** [comment]
> @kacperkurek Thanks you so much :) But could you have not included the linting changes in this PR, to limit changes under review?

Ok I will revert lint changes and make another pr with them, but be aware that tests will have error, that I added lint fix

**Comment by kacperkurek** [comment]
@rzadp Reverted lint changes

**Review by rzadp — APPROVED** [review]

**Comment by rzadp** [comment]
This is so weird that the lint starts to fail on this PR 🤔

**Comment by rzadp** [comment]
There seems to be a lot of changes in the lockfile, I've recreated a PR because I don't have write access to it.
Compare to my changes to the lockfile: https://github.com/TrueFiEng/useDApp/pull/767
I'm putting you as a co-author.

**Pull request closed without merging**